src/main/java/fast_reset/client/mixin/OptionsScreenMixin.java

```diff
@@ -19,23 +18,23 @@ protected OptionsScreenMixin(Text title) {
         super(title);
     }
 
-    private static Text getButtonText(){
-        switch(Client.buttonLocation){
+    private static String getButtonText() {
+        switch (FastReset.buttonLocation) {
             case 0:
-                return new LiteralText("bottom right");
+                return "bottom right";
             case 1:
-                return new LiteralText("center");
+                return "center";
             case 2:
             default:
-                return new LiteralText("replace s&q");
+                return "replace s&q";
         }
     }
```

Comment:
whats the reasoning behind switching this from returning a text object to returning a string?

The better change here if we would want to change it to be better is make it return translatable text

Reply:
returning string seemed easier because it means you only have one spot in the code where it says `new LiteralText()`, not really a necessary change honestly just something i did while i was thinking about other stuff

Reply:
A better refactor then would prob be something like this:
```java
string text;
switch(FastReset.buttonLocation){
    case 0:
        text = "bottom right";
        break;
    case 1:
        text = "center";
        break;
    case 2:
    default:
        text = "replace s&q";
        break;
}
// return new TranslatableText(text);
return new LiteralText(text);
```

Reply:
sounds good will change that